### DISCUSSION TITLE
Aliases in `@include`

### DISCUSSION THREAD

**Discussion by Daeraxa** [vuepress-theme-hope/vuepress-theme-hope#2385] [Q&A] [answered]
This might be a silly question with an obvious answer, in a project I have .md files imported as a package from github (i.e. they are in `.../node_modules/myrepo/`).

What I originally attempted was to create an alias (similar to what is already being used for markdown images) in my `config.js`:

```
alias: {
    '@mydocs': path.resolve(__dirname, '../../node_modules/myrepo')
},
```

Then inside my index.md file for a section I wanted to use `@include` to import those files.

e.g.
```
---
title: My Page
---

A files to blah blah blah....

@include(@mydocs/file1.md)
@include(@mydocs/file2.md)
```
This errors because it seems the entire string is being passed to the include function:

`Include: /home/.../mywebsite/docs/docs/mypage/@mydocs/file1.md not found`

Obviously it works if I reference the file directly:

`@include(../../../node_modules/myrepo/file1.md)`

But it would be useful to not have to put the relative path in like that and just have a single alias that always works.

I also don't want to modify the global include behaviour because other files are being used correctly when placed within the normal `docs` structure.

Is this possible to do?

**Comment by Mister-Hope** [comment, marked as answer]
~~It is possible to add alias support. We will add this in next version.~~

Edited: https://vuepress-theme-hope.github.io/v2/md-enhance/guide/include.html#advanced

We already provide option for you

**Reply by Daeraxa** [reply]
Thank you so much, I did see that option but I wasn't certain it applied to this situation, I think I tried to use it before and made a mistake with it and assumed it was for a different purpose.
I've got it working now, thanks so much for the theme and quick responses.
